src/test/java/org/galatea/starter/service/SettlementServiceTest.java

```diff
@@ -28,14 +29,21 @@
   @MockBean
   private ISettlementMissionRpsy mockSettlementMissionRpsy;
 
+  @MockBean
+  private HistoricalPriceDBRepository rpsy;

```

Comment:
don't create things you aren't using. this file shouldn't be modified

Reply:
This should be resolved now.